src/shared/constants/config/routerPaths.ts

```diff
@@ -105,8 +105,8 @@ const pathsMap = {
   discoveryLanding: () => '/',
   discoveryLeaderboard: () => `/${PathName.leaderboard}`,
   discoveryMyProjects: () => `/${PathName.myProjects}`,
-  discoveryActivity: () => `/${PathName.activiy}`,
-  discoveryGrants: () => `/${PathName.grants}`,
+  discoveryActivity: () => `/${PathName.activity}`,
+  discoveryGrants: (grantId?: string) => (grantId ? `/${PathName.grants}/${grantId}` : `/${PathName.grants}`),
```

Comment:
create a separate one for grantPage, these are supposed to be simple oneOff routes, no logic.

Reply:
This is how it was implemented, I wondered why it was made this way but I agree that there is no reason for it to be different from the other routes.